### PR TITLE
fix(typo): Fix pronouns in hai first contact

### PR DESCRIPTION
**Pull request by tibetiroka**
**Bugfix:** The fix we all thought was done eight months ago (thanks @Hurleveur for reporting)

## Fix Details

~~Was MZ xenophobic?~~

`it` -> `they` in hai first contact, uhai first contact and the hai child mission.

I think the hai child mission could benefit from specifying the gender of the child, but that's work for another PR.

Also, does anybody know if this is correct, or should be `they`?

> After you leave your ship and start making your way through the docking area, you notice a Hai running towards you and waving. As they get closer, you realize that it's Nanachi, the Hai pilot who you helped out years ago.

Probably correct, but worth double-checking.

## Testing Done
None. :smug:

### CODE REVIEW

**Review by Hurleveur — APPROVED** [review]
very nice change, as I think it's inappropriate for the narrator to use it since it denotes a lack of respect for those aliens that are clearly sentient
this also takes one further thing apart from the UHai campaign which is nice

**Review by EjoThims — APPROVED** [review]
The last bit you had a question on is, I believe, indeed correct as is, though quite awkward.

Perhaps change it to "you recognize Nanachi" and circumvent the issue entirely?

File: data/hai/unfettered 0 prologue.txt [hunk 1]
```diff
@@ -50,7 +50,7 @@ mission "First Contact: Unfettered"
 					goto everyone
 			
 			label brethren
-			`	It hisses. "They are not true Hai. We are Hai. The unaltered. We were masters of a hundred worlds before human monkeys had learned the use of thumbs."`
+			`	they hiss. "They are not true Hai. We are Hai. The unaltered. We were masters of a hundred worlds before human monkeys had learned the use of thumbs."`
```
Comment (W1zrad) [inline data/hai/unfettered 0 prologue.txt:53]:
```suggestion
			`	They hiss. "They are not true Hai. We are Hai. The unaltered. We were masters of a hundred worlds before human monkeys had learned the use of thumbs."`
```

**Review by W1zrad — CHANGES_REQUESTED** [review]

**Comment by W1zrad** [comment]
Looked at this on-stream. Our findings are that other species (in cases where gender is unspecified) use `it`, not `they`. As such, I think it makes sense to leave this as-is: with `it`. If we want to change this precedent, let's create a PR encompassing all races, not just the Hai.

**Comment by Hurleveur** [comment]
Other species? We havent met any other sentient species so surely there isnt a rule for them yet?
I'm definitely for doing it for all species, let's *not* be xenophobic, I don't think that is in the spirit of this game

**Comment by EjoThims** [comment]
I don't agree that this should have been closed just because it's not applying it to the whole game.

Not only is it easier to spot such things with a specific focus to certain areas, it allows for more in depth discussion of how, if at all, to appropriately adjust or vary the approach to the change. The questioned passage in the PR here is a perfect example.

Unless the intent is to clearly and distinctly say the current pattern is one we want to maintain (which will probably necessitate PRs in the other direction for consistency anyway), I think this should be reopened and (after continuing review) merged based on its appropriateness to the Hai.

If that is the intent, it also does still need that clear and distinct statement to that effect.

**Comment by tibetiroka** [comment]
About the Hai, we should probably ask @MasterOfGrey.

**Comment by tibetiroka** [comment]
Based on a previous conversation on this topic, I was under the impression tie use of `they` was not a controversial topic. Since that is apparently not the case, should I open a github discussion? Or is it enough to just ask the faction authors?

**Comment by MasterOfGrey** [comment]
In my personal thinking: When it's content where you're new to an alien species, it makes a degree of sense that you don't necessarily view them as 'people' yet - and so 'it' in things like first contact missions and such seem fine to me.
However, once you've established some sort of rapport with an alien race, then I would expect that 'they' makes more sense unless you're deliberately choosing to denigrate them.

Certainly the HR stuff uses 'they' for uncertain Hai, but that whole story deals with experiencing a lot of very 'people' situations with them.

**Comment by Hurleveur** [comment]
By that logic only the first contacts should have it in them then, right?

**Comment by W1zrad** [comment]
Sorry for the confusion, y'all - xenophobia was certainly not my intent.

> We havent met any other sentient species so surely there isnt a grammar rule for them yet?

@Hurleveur to clarify, we looked at Korath, Quarg, and Coalition on-stream. In cases where gender was unspecified, the conclusion was that using `it` rather than `they` seems to be the norm, regardless of familiarity with the species (albeit it was mentioned that Quarg may not have gender, so they may not be the best example). Does something make the Hai's case different from the others?

> I don't agree that this should have been closed just because it's not applying it to the whole game.

@EjoThims sure, I can understand that. My perspective when I closed the PR was that the question of `it` vs. `they` is a larger discussion than what this PR encompasses. As such, it makes more sense (to me) to create an issue that's more geared to the question as a whole instead of having the deliberation be in a PR about one case of the conversation. In that regard, I probably should have suggested that in my closing comment, so that was my bad.

> Based on a previous conversation on this topic, I was under the impression the use of they was not a controversial topic.

@tibetiroka yes, doesn't need to be controversial. I disagreed with the change on the basis that it doesn't match existing precedent, but I didn't really consider the dehumanizing implication of using `it` instead of `they` - which I probably should have.

So if you all want to reopen and keep discussing here (and if the consensus is that `they` should be used instead of `it`), you're welcome to do that, and I'll be happy to merge! Personally, I think it would be cleaner if changes to all races are made in one fell swoop, but "cleanliness" doesn't matter all that much.

**Comment by tibetiroka** [comment]
> Personally, I think it would be cleaner if changes to all races are made in one fell swoop, but "cleanliness" doesn't matter all that much.

I think I'll reopen this pr to make the discussion more visible and set a proper precedent, then do all the remaining species in a separate pr a couple weeks later when I have more time.

**Comment by Hurleveur** [comment]
> @Hurleveur to clarify, we looked at Korath, Quarg, and Coalition on-stream. In cases where gender was unspecified, the conclusion was that using it rather than they seems to be the norm, regardless of familiarity with the species (albeit it was mentioned that Quarg may not have gender, so they may not be the best example). Does something make the Hai's case different from the others?

I understand, and I think we were aware of that, I just thought you meant you looked outside of ES, like online or asking a grammar expert, I dont know.
No, ofc they're not different, it's just my focus as a content creator, and thats where we noticed it first.

**Comment by tibetiroka** [comment]
> where you're new to an alien species, it makes a degree of sense that you don't necessarily view them as 'people' yet

Where does one draw the line? I suppose if you see a giant mantis you may not give it special treatment, but if you see them crew a ship and hand you a hologram projector, it is fair to assume they are a highly intelligent species.

I think if a giant squirrel was talking perfect `<language>` to me, I'd be pretty quick to recognize them as such ~~or quickly admit myself to a mental hospital~~.

Essentially I'm saying that in my opinion, only the first couple lines of the first contact missions could use `it`, everything else should be `they`.

**Comment by MasterOfGrey** [comment]
> Where does one draw the line? 

In general I would say, at about the point where you establish information flow in both directions (however basic it may be).

So to use the current wanderer's setup as an example, I'd say `it` on any landings before it becomes available, and during the first contact right up until the point where you realise they're speaking Hai and can act on that information. <- That is the point where you gain some _actionable_ information directly from an effort that they have taken, rather than indirectly through your own observations.

For most first contacts, this is indeed after only the first few lines.

I'm basing this on the switch that I have observed people seem to subconsciously make when first encountering and being introduced to new animals, particularly pets.

I'm not particularly pressed about this though. If we choose to go 100% one way or another, I'd vote for `they` rather than `it`.
(Unless Quarg truly are `it`s, in which case special case for Quarg?)

**Comment by tibetiroka** [comment]
> (Unless Quarg truly are `it`s, in which case special case for Quarg?)

Where does this come from? I recall some lore about the possibility of the Pug not being individuals / having some sort of group consciousness, but not about the Quarg.

The Drak/Archon might be an interesting one, though.

**Comment by EjoThims** [comment]
I think it's more that Pug either lack (or intentionally de-emphasize personal identity), not that they have group consciousness.

While the Quarg angle seems to be a reference to their lack of gender (though I myself don't recall where that may have come from, this isn't the first I've seen it mentioned either).

Even with that angle, though, (and despite my own personal preferences), I'd still suggest 'they' for Quarg, unless we're going to explicitly have them tell us they prefer otherwise (as well as why, if the right options are chosen). That all, perhaps, is best hashed out in either a discussion or the later PR either dealing with them specifically or other species in general.

I do like Grey's suggestion on the switchover line, though, especially as a literary device to show the shift in humanization based on response.

To me, that is that only remaining change, if any, this PR will need. :D

**Comment by W1zrad** [comment]
@tibetiroka want to change the first 1-2 instances to `it` instead of `they`? If not then this is good to merge.

**Comment by tibetiroka** [comment]
Yes, I'll change them.

**Comment by tibetiroka** [comment]
I changed the first couple instances in unfettered first contact (in the branch where FL doesn't know the Hai yet), but not in hai first contact, because the first time the Hai are addressed with singular pronouns is _after_ they spoke to FL.

**Review by W1zrad — APPROVED** [review]

**Pull request merged**